### DISCUSSION TITLE
Where does weather connect to

### DISCUSSION THREAD

**Discussion by Githubismalware** [fastfetch-cli/fastfetch#1193] [General]
Hello, I found out about the weather module that connects to the internet to find out about weather in my location. However, I care about my privacy, and want to know what url does it contact, because e.g. if it's a link to google more like SNOOPle, that would be very undesirable.

I tried searching through the code to find the url but ultimately only know that the result is hiding in path.chars. If you can reveal the url contacted I would be thankful!

**Comment by CarterLi** [comment]
[wttr.in](https://wttr.in/)

https://github.com/fastfetch-cli/fastfetch/blob/dev/src/detection/weather/weather.c#L22

**Comment by CarterLi** [comment]
> I found out about the weather module that connects to the internet to find out about weather in my location.

No one can magically know the weather in your location without asking the internet
